README.md

```diff
@@ -241,9 +241,19 @@ Pilot instance in the ingress namespace. This will be fixed in future releases.
 Note that running a dedicated Pilot for ingress/gateways is supported and recommended for very large sites, 
 but in the case of K8S ingress it is currently required.
 
+```bash
```

Comment:
we added:
```
./bin/iop istio-ingress istio-ingress-pilot $IBASE/istio-control/istio-discovery --set global.istioNamespace=istio-control-master --set global.configNamespace=istio-control-master --set global.telemetryNamespace=istio-telemetry
```

to add another pilot, but the pilot won't start

```
Error receiving MCP resource: rpc error: code = Unavailable desc = no healthy upstream
```

What is the missing upstream in this case?

Reply:
I think I need more logs - and some debugging. 
But I would suspect the MCP certificates. I only tested pilot in ingress namespace with galley off ( the goal
was to test the most minimal possible config )